osquery/tables/applications/posix/prometheus_metrics.h

```diff
@@ -0,0 +1,53 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+#pragma once
+
+#include <chrono>
+#include <map>
+#include <string>
+#include <vector>
+
+#include <osquery/tables.h>
+
+#include <boost/network/protocol/http/client.hpp>
```

Comment:
nit: We try to group header includes from broadest to most specific, so throw this above the `osquery/tables` include you have above